tests/validation/validate_ball_and_stick.cpp

```diff
@@ -21,66 +21,75 @@
 
 using namespace nest::mc;
 
-// TODO: further consolidate common code
+struct sampler_info {
+    const char* label;
```

Comment:
Only for simplicity: lots of `const char*` literals, and the struct is only used within this one source file, so it didn't need to be more generic.